app/src/main/java/ju/video/player/model/request/AudioRequest.java

```diff
@@ -45,30 +47,36 @@ public AudioRequest () {
      * It is responsible for constructing the body of the request and executing the request.
      * @param path It is the directory where the file to be converted is located.
      * @param format Is the format to which the file is to be converted.
-     * @throws IOException
+     * @throws AudioRequestException
      */
     @Override
-    public void sendPost(String path, String format) throws IOException {
+    public void sendPost(String path, String format) throws AudioRequestException {
         this.path = path;
         this.newFormat = format;
-        HttpClient httpclient = new DefaultHttpClient();
-        HttpPost httppost = new HttpPost(HTTP_POST);
-        FileBody bin = new FileBody(new File(path));
-        StringBody bitrate = new StringBody("");
-        StringBody channels = new StringBody("");
-        StringBody samplingFrequency  = new StringBody("");
-        StringBody outFormat = new StringBody(newFormat);
+        try {
+            validateFormat();
+            validatePath();
+            HttpClient httpclient = new DefaultHttpClient();
+            HttpPost httppost = new HttpPost(HTTP_POST);
+            FileBody bin = new FileBody(new File(path));
```

Comment:
"bin", explain what does mean this variable in you declaration. We need to declare that the variables are significant.